Nikita_Konkov_Application/CSharpOOP3/CSharpOOP3/Ball.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP3
+{
+    internal class Ball: Shape
+    {
+        internal string Name;
+        internal double Radius;
+
+
+        public Ball(string name, double radius): base(name)
+        {
+            Name = name;
+            Radius = radius;
+        }
+
+
+        public override double GetVolumeShape()
+        {
+            double volumeBall = (1.33333 * Math.PI) * Math.Pow(Radius, 3);
```

Comment:
It would be better to use full formula ((4 * Math.PI * Math.Pow(Radius, 3)) / 3).
Read about Math.Round.

Reply:
Corrected it.